.vib/apache/goss/apache.yaml

```diff
@@ -0,0 +1,49 @@
+file:
+  # Main Apache config file includes other config files
+  {{ .Vars.root_dir }}/apache/conf/httpd.conf:
+    exists: true
+    filetype: file
+    contains:
+      - /^Include.*{{ .Vars.root_dir }}/apache/conf/bitnami/bitnami.conf/
+      - /^Include.*{{ .Vars.root_dir }}/apache/conf/vhosts/\*.conf/
+      - /^Include.*{{ .Vars.root_dir }}/apache/conf/deflate.conf/
```

Comment:
Not sure if it's important to check this.

Reply:
If it’s important for our container configuration, we should

Reply:
Yes! This is the main apache configuration file, in which we are performing these changes to include our own configuration files. You can see the explicit logic here:

https://github.com/bitnami/containers/blob/main/bitnami/apache/2.4/debian-11/rootfs/opt/bitnami/scripts/apache/postunpack.sh#L63

This, together with the `check-configuration` test, will ensure that our customizations are included + there are no syntax errors 😁

Reply:
ok, didn't see this before.